20th/articles/rfc22.md

```diff
@@ -0,0 +1,45 @@
+# RFC22: Control flow: Builtin switch statement, by Damian Conway
+
```

Comment:
Please remember to link the RFC somewhere in the body

Reply:
D'oh!